test/typechecker_tests.erl

```diff
@@ -211,11 +211,11 @@ glb_test_() ->
 
      %% Maps
      ?glb( ?t(map()), ?t(#{a := integer()}), ?t(#{a := integer()}) ),
-     %?debugVal( deep_normalize(element(1, glb(?t(#{ a := integer() }), ?t(#{ b := float() })))) , 1000),
-     %?debugVal( deep_normalize(element(1, glb(?t(#{ b := float() }), ?t(#{ a := integer() })))) , 1000),
-     %?debugVal( deep_normalize( ?t(#{ a := integer(), b := float() }) ) , 1000),
-     ?glb( ?t(#{ a := integer() }), ?t(#{ b := float() }),
+     ?glb( ?t(#{ a := integer() }), ?t(#{ b := float() }), ?t(none()) ),
+     ?glb( ?t(#{ a := 1 }), ?t(#{ a := integer() }), ?t(#{ a := 1 }) ),
+     ?glb( ?t(#{ a := integer(), _ => _ }), ?t(#{ b := float(), _ => _ }),
            ?t(#{ a := integer(), b := float() }) ),
```

Comment:
Should the result contain `_=>_` here, i.e. should it be `#{ a := integer(), b := float(), _ => _ }`?

Reply:
Indeed, it seems so when I think about it again. Marking as TODO.